packages/config/__test__/Config.test.js

```diff
@@ -77,6 +75,29 @@ describe('Config - create config with schema', () => {
     expect(() => new Config(schema, configs)).toThrow('"teams.name" must be a string');
   });
 
+  it('should throw error if setSchmea is not Joi', () => {
```

Comment:
Should throw an error if on bad Joi validation(test until row 94 include)
Also from row 95 you can split to a successful test